### DISCUSSION TITLE
Why element is not dropping where i want (below the content)

### DISCUSSION THREAD

**Discussion by gxanshu** [GrapesJS/grapesjs#5239] [Q&A]
i'm working on a application with grapesjs and set drag & drop to True for all blocks
with `data-gjs-draggable=true` `data-gjs-droppable=true` and this works. every block is dragging and dropping in all the elements in the canvas. now the problem is i want to add new block below of previous block but grapresjs not dropping that there. it only drop at the top of the previous block.

here is the example. in this example i have a Row component and inside  that row i have two columns each column cotains custom blocks. i want to append two new columns bellow the previous content but it only append in the uper part 

https://github.com/GrapesJS/grapesjs/assets/68537640/3ecd281b-8a76-49e0-9b8a-58c6562a877f

**Comment by ronaldohoch** [comment]
Can you provide a jsfiddle sample with the bug?

**Reply by ronaldohoch** [reply]
Using this link: https://jsfiddle.net/szLp8h4n
